cirq/sim/clifford/clifford_gate_decomposer.py

```diff
@@ -0,0 +1,73 @@
+from typing import (Tuple)
```

Comment:
There is already a _decompose_into_cliffords method in pauli_string.py that uses the _decompose_into_clifford_with_qubits_ method. Can these be unified so that there is one right way to do a clifford decomposition?

Reply:
_decompose_into_cliffords doesn't dodecomposition itself. It check sthat `op` is already a clifford gate, or it defines `_decompose_into_clifford_`.

In this class I am solving specific problem - given 2x2 unitary, check whether it's a Clifford unitary, and if so - decompose it into produt of H and S gates, and global phase. I didn't find this problem solved (for all Clifford gates) in Cirq. So, I think that this class should exist to solve this problem, and called from other places where this decomposition is needed.